kola/tests/docker/docker.go

```diff
@@ -187,6 +187,30 @@ systemd:
 		Distros: []string{"cl"},
 	})
 
+	register.Register(&register.Test{
+		Run:         func(c cluster.TestCluster) { testDockerInfo("devicemapper", c) },
+		ClusterSize: 1,
+		Name:        "docker.devicemapper-storage",
+		// This test is normally not related to the cloud environment
+		Platforms: []string{"qemu", "qemu-unpriv"},
+		// Note: copied verbatim from https://github.com/coreos/docs/blob/master/os/mounting-storage.md#creating-and-mounting-a-btrfs-volume-file
+		// Added explicit btrfs driver selection because overlay2 is the default for btrfs FS in docker 23 and above
```

Comment:
The comment should rather mention devicemapper, not btrfs2, no?

Reply:
That's correct - fixed.